docs/images.md

```diff
@@ -44,7 +44,7 @@ The tooling used to build these images is open source:
   used for building the kernel.
 
 The latest image name is kept in the [stable channel manifest](https://github.com/kubernetes/kops/blob/master/channels/stable),
-but an example is `kope.io/k8s-1.4-debian-jessie-amd64-hvm-ebs-2016-10-21`.  This means to look for an image published 
+but an example is `kope.io/k8s-1.4-debian-jessie-amd64-hvm-ebs-2016-10-21`.  This means to look for an image published
```

Comment:
The diff is shown here because my editor removed trailing whitespace.

With reference to this - https://github.com/kubernetes/kops/blob/master/channels/stable, for 1.4, this is the latest build.